src/pages/instances/forms/CreateImageFromInstanceSnapshotForm.tsx

```diff
@@ -30,14 +31,17 @@ const CreateImageFromInstanceSnapshotForm: FC<Props> = ({
   const eventQueue = useEventQueue();
   const toastNotify = useToastNotification();
   const queryClient = useQueryClient();
+  const snapshotLabel = (
+    <ResourceLabel bold type="snapshot" value={snapshot.name} />
```

Comment:
I think this should be a chip with a link to instance detail page, snapshots section.